src/sdam/description/topology/test/rtt.rs

```diff
@@ -21,13 +21,15 @@ pub struct TestFile {
 #[serde(untagged)]
 pub enum AverageRtt {
     F(f64),
+    I(i32),
     S(String),
 }
 
 async fn run_test(test_file: TestFile) {
     let avg_rtt_ms = match test_file.avg_rtt_ms {
         AverageRtt::F(f) => Some(f),
         AverageRtt::S(ref s) if s == "NULL" => None,
+        AverageRtt::I(i) => Some(i as f64),
```

Comment:
Our updated extjson parser requires the ".0" suffix to read in an integer as a double, so I added an i32 case here so that these tests would work.